versions/schematic-3.md

```diff
@@ -0,0 +1,222 @@
+
+##### Fields
+
+Field Pattern | Type | Description
+---|:---:|---
+<a name="blockEntityPos"></a>Pos | `integer[3]` | **Required.** The position of the `BlockEntity` relative to the `[0, 0, 0]` position of the schematic (without the [offset](#schematicOffset) applied). Must contain exactly 3 integer values.
+<a name="blockEntityId"></a>Id | `string` | **Required.** The id of the `BlockEntity` type defined by this `BlockEntity` Object, specified as a [Resource Location](#defResourceLocation). This should be used to identify which fields should be required for the definition of this type.
+<a name="blockEntityExtra"></a>Extra | `extra` | **Optional** The extra information related to a `BlockEntity` Object that otherwise would define various bits of information for the `BlockEntity`. This can be formatted in any which way, and likely can change between Minecraft Data Versions.
+
+##### BlockEntity Object Example
+
+An example of possible storage of a sign. See the [Minecraft Chunk Format](http://minecraft.gamepedia.com/Chunk_format#Block_entity_format) for a complete listing of data used to store various types of block entities present in vanilla minecraft. Mods may store additional data or have additional types of block entities.
+
+```json
+{
+    "Pos": [0, 1, 0],
+    "Id": "minecraft:Sign",
+    "Text1": "foo",
+    "Text2": "",
+    "Text3": "bar",
+    "Text4": ""
+}
+```
+
+#### <a name="entityObject"></a>Entity Object
+An object to specify an instance of an [entity type](#defEntityType) within the region. Entities usually occupy a three dimensional space within the region and have many characteristics, varied by the entity type, however the structure will be the same as used by the [Minecraft Chunk Format](https://minecraft.gamepedia.com/Chunk_format#Entity_format), dependent on the `Data Version` the schematic is created from.
+
+##### Fields
+Field Pattern | Type | Description
+---|:---:|---
+<a name="entityPos"></a>Pos | `double`[3] | **Required.** The position of the entity relative to the [0, 0, 0] position of the schematic (without the offset applied). Must contain exactly 3 double values.
+<a name="entityId"></a>Id | `string` | **Required.** The id of the entity type defined by this Entity Object, specified as a [Resource Location](#defResouceLocation). This should be used to identify which fields should be required for the definition of this type.
+<a name="entityExtra"></a>Extra | `unknown` | **Optional** The extra information related to an entity that otherwise is either defaulted or required by the entity associated with the [entity type](#defEntityType)
```

Comment:
```suggestion
<a name="entityExtra"></a>Extra | `extra` | **Optional** The extra information related to an entity that otherwise is either defaulted or required by the entity associated with the [entity type](#defEntityType)
```

Reply:
It’s not determined as the tag in nbt. That’s why it’s unknown.

Reply:
I'm still not certain of what exactly the expected representation here is, it should probably be clarified.